src/views/AddCommunity/index.tsx

```diff
@@ -182,6 +177,25 @@ const AddCommunity: React.FC<{ isLoading?: boolean }> = props => {
                             }
                         }
                     }
+                } else {
+                    try {
```

Comment:
The logic here can be simplified, i think we don't need to duplicate the try/catch block, and if we do, it's better to use an abstraction here. If we don't have image we shouldn't be doing the request, right? because is a required field, or i'm missing something here?

Reply:
the image is only required if the user has no image.
basically what happens is:

- the user fills the form to submit a new community
- if the user is not logged in, it prompts metamask 
- after logged in, the user information form only shows if there's one or more of this atributes missing: image, first name, last name, email
- after the user has all this information filled, the new community can be submitted.

so, to submit a community the user must have image, first name, last name and email and the user info popup only shows the ones missing